features/git-kill/current_branch/on_feature_branch/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,57 @@
+Feature: git kill: killing the current feature branch without a tracking branch (with open changes and without remote repo)
+
+  (see ../without_tracking_branch/with_open_changes.feature)
+
+
+  Background:
+    Given I have a feature branch named "other-feature"
+    And my repo does not have a remote origin
```

Comment:
Actually we can just use `Given I have feature branches named "current-feature" and "other-feature"`.